README.md

```diff
@@ -4,6 +4,29 @@
 
 🔧 Settings for Loadsmart's hosted Danger instance
 
+## How to add Peril to my repo
+
+**Note**: this applies to repositories under Loadsmart organization. If
+you want to install peril for your repositories, you probably want to
+take a look at the general instructions available at
+[peril repository][peril-repo].
```

Comment:
Just to make sure, should this be a link?

Reply:
Yes! There was a typo in the identifier at the end of the file 🤦‍♂️